### ISSUE TITLE
When will a new release that works with Terraform 0.12 be available?

### ISSUE THREAD

**Issue by benjamink** [akamai/terraform-provider-akamai#8]
When will a new release of this provider be made available that works with Terraform 0.12.x?

### Terraform Version
0.12.4

### Expected Behavior
Plugin should work as normal (v0.1.0 works fine with Terraform 0.11).

### Actual Behavior
```
$ terraform apply                                            

Error: Failed to instantiate provider "akamai" to obtain schema: Incompatible API version with plugin. Plugin version: 4, Client versions: [5]
```

**Comment by ElvenSpellmaker** [comment]
I already created #6 which is basically the same as yours =)

**Comment by benjamink** [comment]
This appears to be fixed in the latest 0.1.1 release.

**Comment by ElvenSpellmaker** [comment]
Thanks, I'll close my issue too.
